replace_template.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# This code will take a template file and change it according to the requirements in the integration-definitions repo
+
+file=$1
+integration=$2
+if grep -q "ParameterGroups" "$file"; then
+    yq eval '.Metadata."AWS::CloudFormation::Interface".ParameterGroups[0].Parameters += "IntegrationId"' -i $file
```

Comment:
Are we missing the `--inplace` here? I see it added elsewhere.

Reply:
yes i will add it.